### DISCUSSION TITLE
EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x3800fd1e, pid=1396, tid=12040

### DISCUSSION THREAD

**Discussion by shanewallner73** [winder/Universal-G-Code-Sender#1998] [Help]
i give up 5 hours seveal attempts and nothing
tried java 7 8 732 764 832 864 noting is working and driving me nuts 
thanks in adavnce
Shane

[hs_err_pid1396.log](https://github.com/winder/Universal-G-Code-Sender/files/9689778/hs_err_pid1396.log)
[hs_err_pid11632.log](https://github.com/winder/Universal-G-Code-Sender/files/9689779/hs_err_pid11632.log)
[hs_err_pid12960.log](https://github.com/winder/Universal-G-Code-Sender/files/9689780/hs_err_pid12960.log)
[hs_err_pid764.log](https://github.com/winder/Universal-G-Code-Sender/files/9689781/hs_err_pid764.log)

**Comment by breiler** [comment]
What do you mean by this?
> tried java 7 8 732 764 832 864

There is no need to install Java as it comes bundled with the platform edition. But if you want to run your own JVM you need to specify its location in `etc/ugsplatform.conf`. But that should generally not be needed.

Others that have reported similar errors found that this error was caused by buggy graphics drivers, so make sure that it is updated.
